wgpu/_classes.py

```diff
@@ -196,6 +215,41 @@ def canvas(self):
         """The associated canvas object."""
         return self._canvas_ref()
 
+    def _get_capabilities(self, adapter):
+        """Get dict of capabilities and cache the result."""
+        if self._capabilities is None:
+            self._capabilities = {}
+            if self._present_info["method"] == "screen":
+                # Query capabilities from the surface
+                self._capabilities.update(self._get_capabilities_screen(adapter))
+            else:
+                # Default image capabilities
+                self._capabilities = {
+                    "formats": ["rgba8unorm-srgb", "rgba8unorm"],
+                    "usages": 0xFF,
+                    "alpha_modes": [enums.CanvasAlphaMode.opaque],
```

Comment:
Is this true? Can't we render transparent PNGs?

Reply:
We probably can. But I wanted to leave that for another pr. See also #521